torch/ao/quantization/fx/quantization_patterns.py

```diff
@@ -459,8 +460,11 @@ def modified_load_arg(n: Node):
                             quantized_graph, node_name_to_scope)
                     kwargs = {**self.binary_op_node.kwargs}
                     add_args = (*load_arg(quantized=activation_dtype(qconfig))(self.binary_op_node.args), scale_arg, zero_point_arg)
-                    op = quantized_graph.create_node(
-                        'call_function', self.quantized_binary_op, add_args, kwargs)
+                    # TODO(after initial review): expand to all callsites in this file
+                    op = create_node_from_old_node_preserve_meta(
```

Comment:
can we add to the reference path? this path is going to be deprecated, the ops that's using reference path can be found in https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fx/_lower_to_native_backend.py and  https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fx/quantized_fusion_patterns_and_replacements.py

Reply:
sure.  The current form of this PR is just to align on how this will look.  If we think this way is the way to go, we can add it to all callsites.

Reply:
I dont know the details of FX quantization but the fusion passes (in this example add + relu) will also work? and if so which node's stack trace will be copied.

Reply:
> I dont know the details of FX quantization but the fusion passes (in this example add + relu) will also work? and if so which node's stack trace will be copied.

the simplest thing to do is only preserve stack traces when the new node is created directly from a single old node, which is what this PR implements.  We can discuss what to do about fusions and nodes which do not appear in the original graph such as quant/dequant in a future PR.